### PR TITLE
Clarify "leads" in CFP

### PR DESCRIPTION
**Pull request by cluhmann**
This addresses a comment by @OriolAbril and hopefully clarifies the role of "leads".

### CODE REVIEW

File: content/cfp.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ Each submission will be reviewed by multiple committee members.  The review proc
 
 **Submission outcomes**
 
-The review process will ultimately determine whether each submitted proposal meets the goals of the PyMCon Web Series.  Contributors whose submissions are determined to be acceptable will be contacted.  Subject to scheduling constraints, they will immediately be paired with a PyMCon Web Series lead and begin the process of developing the proposal into a PyMCon Web Series event or given a rough timeline for doing so.
+The review process will ultimately determine whether each submitted proposal meets the goals of the PyMCon Web Series.  Contributors whose submissions are determined to be acceptable will be contacted.  Subject to scheduling constraints, these contributors will immediately be paired with a PyMCon Web Series lead and together will work to turn the proposal into a full-blown PyMCon Web Series event (or given a rough timeline for doing so).
```
Comment (canyon289) [inline content/cfp.md:85]:
```suggestion
The review process will ultimately determine whether each submitted proposal meets the goals of the PyMCon Web Series.  Contributors whose submissions are determined to be acceptable will be contacted.  Subject to scheduling constraints, these contributors will immediately be paired with a PyMCon Web Series Event lead and together will work to turn the proposal into a full-blown PyMCon Web Series event (or given a rough timeline for doing so).
```

**Comment by canyon289** [comment]
Proposal to add a paragraph

cc @OriolAbril 
```
### Who are PyMCon Event Leads
The event leads will be long time PyMC community members who are available to help guide and refine accepted proposals and ultimately launch them as Web Series Events. Their role is to be a resource for you to great event. They will help you setup any ended infrastructure that's needed, such as Discourse topics, as well as some backend details such as defining and enforcing the Code of Conduct.
```

**Comment by OriolAbril** [comment]
The extra paragraph sounds good too. Some extra comments:

Line 10 in cfp raw markdown has:

> The PyMCon Web Series consists of events proposed and led by members of the PyMC community.

How about "co-led" instead?

Line 20 is:

> Submitted proposals will be reviewed and each contributor whose proposal is accepted will be teamed with a PyMCon Web Series lead.  Together, the contributor and the lead will work to turn the proposal into a full-blown PyMCon Web Series event, with the lead providing support and guidance along the way.

If we use "Event lead" that would need to be updated. The "contributor and the lead" part also read a bit strange to me.

**Comment by canyon289** [comment]
@OriolAbril make the suggested edits directly to speed things up!

File: content/cfp.md (hunk 1 above)
Comment (canyon289) [reply content/cfp.md:85]: Committing these changes so we can see Oriols proposal

**Comment by cluhmann** [comment]
Looking to go me.

**Review by OriolAbril — APPROVED** [review]

**Comment by canyon289** [comment]
Ship it

**Pull request merged**